distribution/ddf-common/src/main/resources/etc/registry/registry-custom-slots.json

```diff
@@ -1,45 +1,383 @@
+      "value": [],
       "type": "string"
     },
-    "bindingType": {
+    {
+      "key": "bindingType",
       "displayName": "Service Binding Type",
       "description": "The binding type for the service.",
-      "values": [],
+      "value": [],
       "type": "string"
     }
-  },
-  "Content": {
-  }
+  ],
+  "Content": [
+    {
+      "key": "Name",
+      "displayName": "Content Name",
+      "description": "Name for this metadata content",
+      "value": [],
+      "type": "string",
+      "required": true,
+      "isSlot": false
+    },
+    {
+      "key": "Description",
+      "displayName": "Content Description",
+      "description": "Short description for this metadata content",
+      "value": [],
+      "type": "string",
+      "isSlot": false
+    },
+    {
+      "key": "objectType",
+      "displayName": "Content Object Type",
+      "description": "The kind of content object this will be. Default value should be used in most cases.",
+      "type": "string",
+      "value": "urn:registry:content:collection",
+      "multiValued": false,
+      "isSlot": false,
+      "advanced": true
+    }
+  ],
+  "PersonName": [
+    {
+      "key": "firstName",
+      "displayName": "First Name",
+      "description": "First name",
+      "value": [],
+      "type": "string",
+      "isSlot": false
+    },
+    {
+      "key": "lastName",
+      "displayName": "Last Name",
+      "description": "Last name",
+      "value": [],
+      "type": "string",
+      "required": true,
+      "isSlot": false
+    }
+  ],
+  "TelephoneNumber": [
+    {
+      "key": "phoneType",
+      "displayName": "Phone Type",
+      "description": "Phone type could be work, home, mobile etc.",
+      "type": "string",
+      "isSlot": false
+    },
+    {
+      "key": "countryCode",
+      "displayName": "Country Code",
+      "description": "Country code, i.e. USA=1",
+      "type": "number",
+      "value": 1,
+      "isSlot": false,
+      "inlineGroup": "inline-small"
+    },
+    {
+      "key": "areaCode",
+      "displayName": "Area Code",
+      "description": "Three digit Area Code",
+      "type": "number",
+      "isSlot": false,
+      "inlineGroup": "inline-small"
+    },
+    {
+      "key": "number",
+      "displayName": "Phone Number",
+      "description": "Seven digit phone number. Dots and dashes allowed",
+      "type": "string",
+      "required": true,
+      "isSlot": false,
+      "inlineGroup": "inline-large",
+      "regex":  "/^[0-9]{3}[\\s-\\.,]?[0-9]{4}$/"
+    },
+    {
+      "key": "extension",
+      "displayName": "Extension",
+      "description": "Extension",
+      "type": "number",
+      "isSlot": false
+    }
+  ],
+  "EmailAddress": [
+    {
+      "key": "type",
+      "displayName": "Email Type",
+      "description": "Email type could be work, personal, primary etc.",
+      "value": [],
+      "type": "string",
+      "isSlot": false
+    },
+    {
+      "key": "address",
+      "displayName": "Address",
```

Comment:
This is under the Email Address section so that is why i when with just address. If you think it needs to be email address let me know.